scripts/dts/gen_defines.py

```diff
@@ -862,6 +862,9 @@ def write_global_macros(edt):
     out_dt_define("FOREACH_OKAY_HELPER(fn)",
                   " ".join(f"fn(DT_{node.z_path_id})" for node in edt.nodes
                            if node.status == "okay"))
+    out_dt_define("FOREACH_OKAY_VARGS_HELPER(fn, ...)",
```

Comment:
You must update doc/guides/dts/macros.bnf whenever you add new generated macros.

Reply:
okay, will update it.